### PR TITLE
add fallback message for various content

### PR DESCRIPTION
**Pull request by maggie-lin**
**Why**
For certain content that isn't being shown at this time (videos, large tables), we can let customers know they can view their content in the Guru app. See [CH ticket](https://app.clubhouse.io/guru/story/32342/add-fallback-message-for-various-content-in-ms-teams) and [messaging](https://docs.google.com/spreadsheets/d/1U6vywuTWXtTbmAW86U3YP_qPCS6RGikNseQKiCBr_kI/edit#gid=0) 

**What**
- add fallback message for `<iframe>` content in emphasized block (different message for video vs. non-video)
- wrap fallback message for large tables in emphasized block
- allow `<div>` and `<iframe>` to be void elements (they usually are in Guru)
- everything in `dist/` is a repeat of everything else

### CODE REVIEW

File: dist/adaptive-html.es.js [hunk 1]
```diff
@@ -380,10 +380,19 @@ rules.strong = {
     });
   }
 };
+rules.iframe = {
+  filter: 'iframe',
+  replacement: function replacement(content, node) {
+    var fallbackText = 'To view this content, please open this video in the Guru app';
```
Comment (RamanjitKhakh) [inline dist/adaptive-html.es.js]: should this say something like `please open this iframe in the Guru app` vs video?

File: tests/to-json-tests.js [hunk 2]
```diff
@@ -927,4 +926,22 @@ test('can handle guru code block', t => {
...
+            style: "emphasis",
+            items: [{
+                type: "TextBlock",
+                text: "To view this content, please open this video in the Guru app",
+                wrap: true
+            }]
+        }],
+        actions: [],
+        version: expectedVersion
+    });
```
Comment (RamanjitKhakh) [inline tests/to-json-tests.js:946]: is there a test for divs as well?

File: dist/adaptive-html.cjs.js [hunk 3]
```diff
@@ -382,10 +382,19 @@ rules.strong = {
...
+      style: 'emphasis'
+    });
+  }
+};
 rules.image = {
   filter: 'img',
   replacement: function replacement(content, node) {
-    var alt = node.alt || '';
+    var alt = node.getAttribute('alt') || '';
     var src = node.getAttribute('src') || '';
```
Comment (austinparkk) [inline dist/adaptive-html.cjs.js:404]: are these values ever changing?

**Review by austinparkk — APPROVED** [review]

File: dist/adaptive-html.es.js (hunk 1 above)
Comment (maggie-lin) [reply dist/adaptive-html.es.js]: fair question, i'll ask Andrew since he came up with the messaging

File: dist/adaptive-html.cjs.js (hunk 3 above)
Comment (maggie-lin) [reply dist/adaptive-html.cjs.js:404]: nope! i can update them to `const` in `adaptivecard-rules.js`

File: tests/to-json-tests.js (hunk 2 above)
Comment (maggie-lin) [reply tests/to-json-tests.js:946]: yeah there are a couple div tests already, let me rename them for clarity

File: dist/adaptive-html.es.js (hunk 1 above)
Comment (maggie-lin) [reply dist/adaptive-html.es.js]: yeah good call - gonna use a more generic message for non videos

**Review by RamanjitKhakh — APPROVED** [review]

**Pull request merged**